model-optimizer/mo/utils/ir_engine/ir_engine.py

```diff
@@ -404,3 +414,73 @@ def __eq__(self, other):
         if not isinstance(other, IREngine):
             raise AttributeError("IREngine can be compared only with IREngine object type")
         return self.compare(other)[0]
+
+    def __read_if(self, layer, layer_attrs):
```

Comment:
@achetver , please, take a look at these changes.

Reply:
We look at these changes together with @evolosen several weeks ago and all works fine.